resources/views/settings/cards/webhooks.blade.php

```diff
@@ -0,0 +1,47 @@
+@php use Carbon\Carbon; @endphp
+<div class="card mt-3">
+    <div class="card-header">{{ __('settings.title-webhooks') }}</div>
+    <div class="card-body">
+        <p>
+            {{ __('settings.webhook-description') }}
+        </p>
+        @if(count($webhooks) == 0)
+            <p class="text-danger">{{__('settings.no-webhooks')}}</p>
+        @else
+            <table class="table table-responsive">
+                <thead>
+                    <tr>
+                        <th>{{ __('settings.client-name') }}</th>
+                        <th>{{ __('settings.created') }}</th>
+                        <th>{{ __('settings.webhook-event-notifications-description') }}</th>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    @foreach($webhooks as $webhook)
+                        <tr>
+                            <td>{{ $webhook->client->name }}</td>
+                            <td>{{ Carbon::parse($webhook->created_at)->isoFormat(__('datetime-format')) }}</td>
+                            <td>
+                                <ul>
+                                    @foreach($webhook->events as $event)
+                                        <li>{{ __('settings.webhook_event.' . $event)}}</li>
+                                    @endforeach
+                                </ul>
+                            </td>
```

Comment:
For some reasons, this doesn't work for me.

<img width="776" alt="image" src="https://user-images.githubusercontent.com/2796271/221356449-765638be-c689-49fa-8ff7-d98710c1dcf0.png">